includes/wikia/transaction/TransactionTraceNewrelic.php

```diff
@@ -38,7 +38,8 @@ function __construct() {
 
 		// report request ID as a custom request parameter for easier debugging of slow transactions
 		if ( function_exists( 'newrelic_add_custom_parameter' ) ) {
-			newrelic_add_custom_parameter( 'requestId', RequestId::instance()->getRequestId() );
+			newrelic_add_custom_parameter( 'requestId', WikiaTracer::instance()->getTraceId() );
```

Comment:
I thought of just a transition period only. I didn't want to break anything that could rely on this but actually it's unlikely something is consuming that information automatically so I could remove it now. Would you agree?